core/Network/TLS/Parameters.hs

```diff
@@ -197,7 +197,8 @@ defaultSupported = Supported
     { supportedVersions       = [TLS12,TLS11,TLS10]
     , supportedCiphers        = []
     , supportedCompressions   = [nullCompression]
-    , supportedHashSignatures = [ (Struct.HashSHA512, SignatureRSA)
+    , supportedHashSignatures = [ (Struct.HashTLS13,  SignatureRSApssSHA256)
```

Comment:
Oh. This is a mistake. I don't want to modify this module.
I modified this file for local testing. I will revert this.